src/DGtal/base/Trace.ih

```diff
@@ -259,7 +259,7 @@ DGtal::Trace::progressBar(const double currentValue, const double maximumValue)
     }
   // and back to line begin - do not forget the fflush to avoid output buffering problems!
 
-  BOOST_STATIC_CONSTANT(std::string, rotation_string = "|\\-/");
+  std::string rotation_string = "|\\-/";
```

Comment:
const std::string ... ?

(I agree that the static stuff is not necessary)

Reply:
yes ;)